### PR TITLE
SBTWO-8575 Added aria-label to close button to appease lighthouse

### PR DESCRIPTION
**Pull request by WarwickGoatee**

### CODE REVIEW

**Review by lol768 — APPROVED** [review]

File: src/StatusPageEmbed.js [hunk 1]
```diff
@@ -272,7 +272,7 @@ class StatusPageEmbed extends Component {
           ) }
         </div>
         <div className="StatusPageEmbed__close">
-          <button className="StatusPageEmbed__close__button" onClick={() => this.dismiss()}>
+          <button className="StatusPageEmbed__close__button" aria-label="Close" onClick={() => this.dismiss()}>
             <FontAwesomeIcon icon={faTimes} />
```
Comment (lol768) [inline src/StatusPageEmbed.js:277]: do we `aria-hidden` this already?

Comment (WarwickGoatee) [reply src/StatusPageEmbed.js:277]: Don't think so

Comment (lol768) [reply src/StatusPageEmbed.js:277]: https://www.456bereastreet.com/archive/201609/hiding_inline_svg_icons_from_screen_readers/ suggests we should, but I suspect `FontAwesomeIcon` is not ours..

Comment (lol768) [reply src/StatusPageEmbed.js:277]: Screenshot_2020-02-14 Font Awesome How to Use

Comment (WarwickGoatee) [reply src/StatusPageEmbed.js:277]: Yeah it's not...

Comment (WarwickGoatee) [reply src/StatusPageEmbed.js:277]: There you go...

**Pull request merged**